tools/metrics/import/import-prometheus-s3.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+usage() {
+    # TODO: Optionally being able to provide a different path to save the files to other than default
+    echo "Usage: $0 [--profile AWS_PROFILE] -f FILE_PATH"
+    exit 1
+}
+
+# Initialize variables
+AWS_PROFILE=""
+FILE_PATH=""
+
+# Parse arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --profile) AWS_PROFILE="$2"; shift;;
+        -f) FILE_PATH="$2"; shift ;;
+        *) usage ;;
+    esac
+    shift
+done
+
+echo "Using AWS_PROFILE=$AWS_PROFILE"
+
+if [[ -z "$FILE_PATH" ]]; then 
+    echo "Need to supply the s3 path to the prometheus data"
+    usage 
+fi
+
+prom_data_dir=$(dirname "$(realpath "$0")")
+
+mkdir -p "$prom_data_dir/temp"
+
+# Delete and recreate the data directory
+if [[ -d "$prom_data_dir/data" ]]; then
+    echo "Data directory exists. Attempting to delete..."
+    sudo rm -rf "$prom_data_dir/data"
+    if [[ $? -ne 0 ]]; then
+        echo "Error: Failed to delete the data directory. Check permissions."
+        exit 1
+    fi
+fi
+
+mkdir -p "$prom_data_dir/data"
+echo "Data directory recreated."
+
+if [[ -z $AWS_PROFILE ]]; then 
+    aws s3 cp "$FILE_PATH" "$prom_data_dir/temp"
```

Comment:
Do we have to provide the whole URI including the scheme ? I would also split `FILE_PATH` variable to differentiate the S3 bucket name from the files it contains. And would even optionalize bucket name provision to point on a hardcoded base bucket by default.

Reply:
There is no hard bucket for now, Quentin has to decide on how to handle things (if we're using separate buckets for metrics/logs/database or a single bucket and then different folders per run with everything related to the run in the same folder)

Reply:
Good point but another point is : do we have to provide the file path like this s3://{bucket-name}/{key-name} ? Wouldn't be very handy